packages/conveyer/src/types.ts

```diff
@@ -10,6 +10,7 @@ import Conveyer from "./Conveyer";
  * @property - scroll direction. (true: Horizontal Scroll, false: Vertical Scroll)  (default: true) <ko>스크롤 방향. (true: 가로 스크롤, false: 세로 스크롤) (default: true)</ko>
  * @property - selector to find items inside. (default: "") <ko>내부의 아이템들을 찾기 위한 selector. (default: "")</ko>
  * @property - Whether to use drag (default: true) <ko> 드래그를 사용할지 여부. (default: true)</ko>
+ * @property - Whether to use the mouse wheel input for same scroll direction (default: true) <ko>스크롤 방향으로 마우스 휠 입력을 사용할지 여부. (default: true)</ko>
```

Comment:
It would be better to use the name useVerticalWheel or useSideWheel.

The explanation seems to be that the scroll direction and the wheel direction are the same.

Reply:
And it may be better to set the default to false.